CRITICAL - 2023-10-10 15:59:08 --> Declaration of App\Models\FormModel::delete() should be compatible with CodeIgniter\BaseModel::delete($id = NULL, bool $purge = false)
in APPPATH\Models\FormModel.php on line 13.
 1 APPPATH\Models\FormModel.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Declaration of App\\Models\\FormModel::delete() should be compatible with CodeIgniter\\BaseModel::delete($id = NULL, bool $purge = false)', 'C:\\wamp\\www\\APP_CODE4\\app\\Models\\FormModel.php', 13, [...])
 2 SYSTEMPATH\Autoloader\Autoloader.php(314): include_once('C:\\wamp\\www\\APP_CODE4\\app\\Models\\FormModel.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp\\www\\APP_CODE4\\app\\Models\\FormModel.php')
 4 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\FormModel')
 5 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\FormModel')
 6 APPPATH\Controllers\Contato.php(25): spl_autoload_call('App\\Models\\FormModel')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Contato->salvar()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-10 16:01:56 --> Error connecting to the database: mysqli_sql_exception: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.
 in C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('borcm8rluzhwfxn...', 'udwwn2u4ioi01ez...', 'IOK0O5JlxmQGfDF...', 'borcm8rluzhwfxn...', 3306, '', 0)
#1 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp\www\APP_CODE4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\wamp\www\APP_CODE4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp\www\APP_CODE4\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp\www\APP_CODE4\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\wamp\www\APP_CODE4\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\wamp\www\APP_CODE4\app\Controllers\Contato.php(32): CodeIgniter\BaseModel->save(Array)
#9 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(932): App\Controllers\Contato->salvar()
#10 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
#11 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\wamp\www\APP_CODE4\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.
 in C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp\www\APP_CODE4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#3 C:\wamp\www\APP_CODE4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\wamp\www\APP_CODE4\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\wamp\www\APP_CODE4\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\wamp\www\APP_CODE4\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#7 C:\wamp\www\APP_CODE4\app\Controllers\Contato.php(32): CodeIgniter\BaseModel->save(Array)
#8 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(932): App\Controllers\Contato->salvar()
#9 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
#10 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp\www\APP_CODE4\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-10 16:01:56 --> Unable to connect to the database.
Main connection [MySQLi]: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_contato` (`nome`, `email`, `telefone`, `mensagem`) VALUES (:nome:, :email:, :telefone:, :mensagem:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\Contato.php(32): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Contato->salvar()
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-10 16:02:00 --> Error connecting to the database: mysqli_sql_exception: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.
 in C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('borcm8rluzhwfxn...', 'udwwn2u4ioi01ez...', 'IOK0O5JlxmQGfDF...', 'borcm8rluzhwfxn...', 3306, '', 0)
#1 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp\www\APP_CODE4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\wamp\www\APP_CODE4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp\www\APP_CODE4\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp\www\APP_CODE4\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\wamp\www\APP_CODE4\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\wamp\www\APP_CODE4\app\Controllers\Contato.php(32): CodeIgniter\BaseModel->save(Array)
#9 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(932): App\Controllers\Contato->salvar()
#10 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
#11 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\wamp\www\APP_CODE4\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.
 in C:\wamp\www\APP_CODE4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp\www\APP_CODE4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp\www\APP_CODE4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#3 C:\wamp\www\APP_CODE4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\wamp\www\APP_CODE4\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\wamp\www\APP_CODE4\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\wamp\www\APP_CODE4\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#7 C:\wamp\www\APP_CODE4\app\Controllers\Contato.php(32): CodeIgniter\BaseModel->save(Array)
#8 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(932): App\Controllers\Contato->salvar()
#9 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
#10 C:\wamp\www\APP_CODE4\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp\www\APP_CODE4\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-10 16:02:00 --> Unable to connect to the database.
Main connection [MySQLi]: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_contato` (`nome`, `email`, `telefone`, `mensagem`) VALUES (:nome:, :email:, :telefone:, :mensagem:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\Contato.php(32): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Contato->salvar()
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Contato))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
